Assets/_BForBoss/_Core/Scripts/EnergySystem/EnergySystemBehaviour.cs

```diff
@@ -84,7 +93,11 @@ private void Start()
 
         private void Update()
         {
-            _elapsedTime -= Time.deltaTime;
+            if (_accruedEnergyTypeQueue.Count > 0 || _expendEnergyTypeQueue.Count > 0)
```

Comment:
`Update` Get's called various times per frame. so the accruement vs disposing extra values get's super unbalanced and off. 

 So i'm changing it where it only counts down when there's something in the queue.